index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <!-- meta data -->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+
+    <!-- page title -->
+    <title>News Blog</title>
+
+    <!-- custom stylesheet -->
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="d-flex col">
+    <header class="d-flex col center">
+        <div class="d-flex col center">
+            <span id="title">News Blog</span>
+            <span id="sub-title">worldwide news</span>
+        </div>
+    </header>
+
+    <main>
+        <section id="articles-header" class="d-flex row">
+            <div class="d-flex col" id="category">
+                <label for="filter">select category</label>
+                <select id="filter">
+                    <option value="All" selected>All</option>
+                    <option value="tech">Technology</option>
+                    <option value="sports">Sports</option>
+                </select>
+            </div>
+            <div id="main-title">All News</div>
+            <div class="d-flex col" id="subscribe-form">
+                <label for="text-subscribe">Subscribe</label>
+                <form action="#!" class="d-flex row">
+                    <input type="email" id="text-subscribe" placeholder="Email Address">
+                    <button type="submit" id="btn-subscribe">Subscribe</button>
+                </form>
+            </div>
+        </section>
+        
+        <section class="d-flex col">
+            <div class="article-container d-flex row broder-bottom">
+                <div class="article-image"></div>
+                <div class="article-body">
+                    <p class="article-title">Title One</p>
+                    <small>Posted on <b>03 Dec, 2021 </b>// Category: <b>Category One</b></small>
+                    <p class="article-desc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos a tempora numquam qui mollitia facilis incidunt necessitatibus eius, aliquam quam aliquid sequi nesciunt error ad sunt esse! Laboriosam, dolore debitis.</p>
+                    <button>Continue Reading</button>
+                </div>
+            </div>
+
+            <div class="article-container d-flex row broder-bottom">
+                <div class="article-image"></div>
```

Comment:
do we really need an empty div here?

Reply:
to display a box with some background

Reply:
Still finding replacement for empty div. Thinking of using img tag.